composer.lock

```diff
@@ -2683,16 +2683,16 @@
         },
         {
             "name": "drupal/core",
-            "version": "8.7.7",
+            "version": "8.7.8",
```

Comment:
Is this expected/wanted in this PR? Unlikely to be a big issue but it's worth taking care we don't increment/update any modules/libraries that aren't strictly connected to the work being undertaken in the ticket.

We can always adjust the version requirements for this in composer.json if it happens routinely.

Reply:
I think I had run composer update last week to try and update the origins module and it must have updated core on my branch. Would it be better if I try to roll it back and add a new commit?